projects/core/src/product/connectors/search/product-search.connector.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable } from '@angular/core';
+import { ProductSearchAdapter } from './product-search.adapter';
+import {
+  ProductSearchPage,
+  SuggestionList,
+} from '../../../occ/occ-models/occ.models';
+import { SearchConfig } from '../../model/search-config';
+import { Observable } from 'rxjs';
+
+const DEFAULT_SEARCH_CONFIG: SearchConfig = {
```

Comment:
This seems to belong to the OCC adapter rather the this generic connector. Can we move it down?

Reply:
👍